src/harness/client.ts

```diff
@@ -314,15 +314,25 @@ export class SessionClient implements LanguageService {
         return notImplemented();
     }
 
-    getNavigateToItems(searchValue: string): NavigateToItem[] {
+    getNavigateToItems(searchValue: string, maxResultCount: number, file: string | undefined, _excludeDtsFiles: boolean | undefined, excludeLibFiles: boolean | undefined): NavigateToItem[] {
         const args: protocol.NavtoRequestArgs = {
             searchValue,
-            file: this.host.getScriptFileNames()[0],
+            file,
+            currentFileOnly: !!file,

```

Comment:
One thought: if I have `excludeLibrarySymbols` enabled, but then I manually open `node_modules/typescript/lib/lib.dom.d.ts` and use ⌘⇧O (symbol search in current file), I still want to see results from the current file.